app/css/components/mentor/discussion.css

```diff
@@ -507,7 +507,7 @@
             & > .c-introducer {
                 @apply mb-20;
             }
-            & > .title {
+            & .title {
```

Comment:
Why this change? This will change any div with the class `.title` in the scratchpad to have this styling, rather than the one this was originally targetting. Is that what you intended?

Reply:
Looking at the CSS, you might be better just inlining this CSS onto the `title` div now.

Reply:
Yes, I moved the title a level deeper. because a level higher it spawned a one pixel invisible top something which only moved the icon 1px down. Unfortunately couldn't figure out why it did that. It still bugs me.. So I moved the title a level deeper, and that solved the icon shift. 

Then I will move the styles into inline classes.

Reply:
I'll check it out and take a look at the 1px thing :)

Reply:
This is a what I used IIRC: (the effect is exactly the same)
```
       <div className="title justify-between">
        <div className="items-center py-[1px] flex">
          Your notes for <strong>{exercise.title}</strong> in
          <TrackIcon iconUrl={track.iconUrl} title={track.title} />
          <strong>{track.title}</strong>
        </div>

        {content === page.contentMarkdown ? null : <AlertTag>Unsaved</AlertTag>}
      </div>

```
you need to add the padding for the div under title, so the height matches the height of the AlertTag.

and you need to remove margin and alignment from AlertTag.tsx: 

```


import React from 'react'

export function AlertTag({
  children,
}: {
  children: React.ReactNode
}): JSX.Element {
  return (
    <div className="text-alert bg-veryLightRed text-13 rounded-8 font-semibold py-6 px-12 animate-fadeIn">
      {children}
    </div>
  )
}
```